lib/structures/Guild.js

```diff
@@ -894,11 +894,11 @@ class Guild extends Base {
 
     /**
     * Get the guild permissions of a member
-    * @arg {String | Member} memberID The ID of the member or a Member instance
+    * @arg {String | Member | Object} memberID The ID of the member or a Member object
     * @returns {Permission}
     */
     permissionsOf(memberID) {
-        const member = memberID instanceof Member ? memberID : this.members.get(memberID);
+        const member = typeof memberID === "string" ? this.guild.members.get(memberID) : memberID;
```

Comment:
this.guild causes an error because you're already in a Guild instance, need to change to .members only